content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -98,8 +130,96 @@ aws configure list
+
+#### Working with O3DE CDK Examples
+If you deploy both the Core and example stack from the AWS Core Gem you can see examples of automatically generated user permissions via the ```generate_user_policy``` and ```generate_admin_policy``` functions, which are then automatically attached to the user groups via the call to ```__grant_access```.
+
+See the CDK [Permissions](https://docs.aws.amazon.com/cdk/latest/guide/permissions.html) documentation for more details.
+
+The AWS feature gems, such as AWS Metrics Gem, during deployment automatically create managed policies for users and admins that can then be attached manually to the appropriate user group. 
+
+##### Attach Managed Policies to User Groups via the AWS Console
+* Open the AWS console at https://console.aws.amazon.com/cloudformation.
+* Navigate to the desired stack in a region.
+* Click the **Resources** tab.
+* Look for resources with the type ```AWS::IAM::ManagedPolicy``` in the stack and record the policy name.
+* Next go to IAM's console at https://console.aws.amazon.com/iam
+* Select **user groups**
+* Click on the user group
+* Select **permissions**, then **add permissions** and select **attach polices**.
+* Filter for policy to attach and select it
+* Hit **Attach Policies**.
+
+##### Attach Managed Policies to User Groups via the AWS CLI
+* List all stacks ```aws cloudformation describe-stacks --region <region>```.
+* [List required stack resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-listing-stack-resources.html) ```aws cloudformation list-stack-resources --stack-name <feature stack> --region <region>```.
+* Search for the ```AWS::IAM::ManagedPolicy```resources and record the physical id of the resource, which should be in the form of the policy ARN.
+* [Attach the relevant policy](https://docs.aws.amazon.com/cli/latest/reference/iam/attach-group-policy.html) to the user groups as desired: ```aws iam  attach-group-policy --group-name <group name> --policy-arn <policy arn>```.
+
+## Setting up Credentials for a Team
+
+{{< note >}}
+You can use [AWS Single Sign-On (SSO)](https://aws.amazon.com/single-sign-on/) instead of IAM to enable multiple users within a single AWS account to work with O3DE. In this usage pattern, the single AWS account serves as the management account for an organization in AWS Organizations, and that organization has no member accounts. To use AWS SSO, skip this topic and follow the instructions in Enterprise Setup instead. For related information, see the [What is AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html) and [What is AWS Single Sign-On](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html) guides.
```

Comment:
"follow the instructions in Enterprise Setup"
Link?

Reply:
Updated instructions (sorry was originally planning an SSO page but its far beyond the scope of the next O3DE release so wasn't included).